dev-db/influx-cli: new package, add 2.7.3

stkw0 opened the pull request.

thesamesam commented on dev-db/influxdb/influxdb-2.7.1.ebuild:
```diff
@@ -180,6 +180,10 @@ COMMON_DEPEND="
 DEPEND="${COMMON_DEPEND}"
 
+RDEPEND="
+	cli? ( dev-db/influx-cli )
```
If we're going to have this as a transition thing, we should default on +cli and also add a warning in pkg_postinst like we did for Docker.

I don't think we should necc. keep this long-term, as it's a runtime-only USE flag (https://projects.gentoo.org/qa/policy-guide/use-flags.html#pg0802).

thesamesam replied:
Yeah, I think that should be OK. It's good enough to help most people.

thesamesam commented:
Could you rebase? William revbumped influxdb.

stkw0 commented:
I updated influx-cli ebuild adding the shell completions. Also I added the licenses of the Go dependencies (if I remember correctly as Go is statically linked the ebuild should contain the licenses of dependencies as well). 

Regarding the influxdb ebuild I am not sure about which is the @williamh approach. I see he is installing a script for the migration so maybe the log message can be removed or changed. Also I saw he didn't add the cli USE flag so maybe he doesn't wants that USE flag? 
I will wait for @williamh input before making further changes to it.
